test/com/facebook/buck/android/testdata/android_instrumentation_test_integration_test/apps/myapp_test/BUCK

```diff
@@ -0,0 +1,14 @@
+android_instrumentation_apk(
+  name = 'app_test',
+  manifest = 'AndroidManifest.xml',
+  apk = '//apps/myapp:app-exo',
+  deps = [
+    '//test/com/example/activity:activity_test',
+  ],
+)
+
+android_instrumentation_test(
+  name = 'test',
+  apk = '//apps/myapp:app-exo',
```

Comment:
this is redundant.. if you're supporting using android_instrumentation_apk() in the instrumentation_apk() target, you can automatically figure this out. So let's remove this altogether:

a. for apks instrumenting itself, we just need to point to an android_binary()
b. for apks instrumenting another apk, we point to an android_instrumentation_apk(). 

Is this easy to do?